### DISCUSSION TITLE
How to get github achivment badges

### DISCUSSION THREAD

**Discussion by artabed** [community/community#18441] [Other Feature Feedback, Questions, & Ideas]
..

**Comment by akbar-ardiansyah** [comment]
By acepted 2 answer

**Reply by artabed** [reply]
How about YOLO badge

**Reply by akbar-ardiansyah** [reply]
Merge without reviews

**Reply by Zerotask** [reply]
It's so obvious that you use 2 accounts and accept your own questions 😂

**Reply by aaronlamz** [reply]
Visit this link https://github.com/orgs/github-community/discussions/19146

**Comment by aaronlamz** [comment]
Here is some more info:
https://github.com/Schweinepriester/github-profile-achievements#achievements
